src/solvers/dgsem_unstructured/sort_boundary_conditions.jl

```diff
@@ -99,6 +99,14 @@ function initialize!(boundary_types_container::UnstructuredSortedBoundaryTypes{N
         _boundary_indices[j] = sort!(indices_for_current_type)
     end
 
+    # Check if all boundaries (determined from connectivity) are equipped with a boundary condition
+    for (index, boundary_name) in enumerate(cache.boundaries.name)
+        if boundary_name ∉ keys(boundary_dictionary)
```

Comment:
```suggestion
        if !(boundary_name in keys(boundary_dictionary))
```

@sloede does not like Unicode